cachito/web/api_v1.py

```diff
@@ -511,6 +520,10 @@ def patch_request(request_id):
         if request.state.state_name == new_state and request.state.state_reason == new_state_reason:
             flask.current_app.logger.info("Not adding a new state since it matches the last state")
         else:
+            if new_state == "complete":
+                cachito_metrics["request_duration"].observe(
+                    (datetime.now() - request.created).total_seconds()
```

Comment:
Ok.  I'll revisit that tomorrow;  It appeared to be behaving properly when I was testing it.  .utcnow() will be the likely alternative....

Reply:
https://github.com/containerbuildsystem/cachito/blob/fe5633ad782b407d99c9243270dd42bc7d91fa85/cachito/web/models.py#L309 -  I think the sqlite behavior isn't UTC as expected, causing my testing to be inaccurate.  I think this needs a 'UTC' modifier, based on the docs here - https://www.sqlite.org/lang_datefunc.html

@sumincho22 -  thoughts?

Reply:
In SQLite, `'now'` provides the UTC datetime by default. For it to be local, you have to add in another parameter with `'localtime'` which we do not have.

Reply:
Ah ha, I see, you're corect.  Too many options in sqlite.  Back to testing for me.

Reply:
Ok, I need the :dark-headdesk:.   Testing with docker-compose, its actually using postgresql.  Anyways, I've figured it all out.

- Yes, the db model is storing (and returning) the datetime in UTC.
- Yes, datetime.now() is returning it in UTC. 

Here's the fun part.  The cachito-api container is set to UTC.  It inherits this from the fedora 35 container its based on.  The /etc/localtime is set to UTC.

Reply:
I understand the cachito-api container is set to UTC timezone when we run it locally and on Jenkins/GitHub, but isn't there a chance that a user may run the cachito-api container on a pod that is not set to UTC timezone? I thought this was the rationale for the UTC datetime issue I did earlier.

Maybe we can move the utcnow functions from cachito/web/models.py to a more general utils.py file, and you can utilize utcnow (supports both PostgreSQL and SQLite) just in case.

Reply:
Only if the Dockerfile used to build it is changed...  as long as its built with that Dockerfile that specifies the Fedora image, it inherits the UTC timezone....